ADAL/src/cache/ADTokenCacheAccessor.m

```diff
@@ -342,52 +359,52 @@ - (void)removeItemFromCache:(ADTokenCacheItem *)cacheItem
         return;
     }
     
-    NSUUID* correlationId = [context correlationId];
+    
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the useless trailing whitespaces at the end of this line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=objc%3AS1131)